README.md

```diff
@@ -254,7 +254,7 @@ Compared to regular install via pip, [requirements.txt](requirements.txt)
 additionally installs developer dependencies such as Cython. For more details
 and instructions, see the documentation on
 [compiling spaCy from source](https://spacy.io/usage#source) and the
-[quickstart widget](https://spacy.io/usage#section-quickstart) to get the right
+[quick-start widget](https://spacy.io/usage#section-quickstart) to get the right
```

Comment:
I'm not sure I want to change this, since we do refer to it as "quickstart" all across the documentation. I just checked and Wikipedia spells it "Quickstart", too, so it seems acceptable? https://en.wikipedia.org/wiki/Quickstart_guide